apps/src/code-studio/components/stageExtras/BonusLevels.jsx

```diff
@@ -83,17 +88,21 @@ class BonusLevel extends React.Component {
   }
 }
 
+const ConnectedBonusLevel = connect((state, ownProps) => ({
+  perfected: isPerfect(state.progress, ownProps.levelId),
+}))(BonusLevel);
+
 export default function BonusLevels(props) {
```

Comment:
Do we still need to export the unconnected version of component?

Reply:
Yeah, this is the component that's actually used in `StageExtras.jsx`. It's just a bunch of `ConnectedBonusLevel`s next to each other with a title.